src/components/AddressPie.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { RadialChart } from "react-vis";
+const AddressPie = ({ addresses = [], ...props }) => {
+  const data = addresses.slice(0, 2).map((a) => ({
```

Comment:
Whats the (0,2)?

Reply:
Returns a new array with only the first two items

Reply:
This is because users will only ahve 2 addresses, but the test user we have has about 8 or 9

Reply:
Cant we make it for any number of addresses?